### PR TITLE
feat(pagination): pagination.js

### LINKED ISSUE #10: Create proper markup of sandbox app
TBD

### PR DESCRIPTION
**Pull request by RhnSaxena**
Fixes #10 (pagination buttons creation part)

### CODE REVIEW

File: static/js/pagination.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+// Function to be called whenever a
+// page no is clicked for pagination
+function paginationClick(page) {
+  fhirSearch(resource, searchParams, page, count);
```
Comment (sid22) [inline static/js/pagination.js:4]: from where will you get `resource, searchParams, count` here?

File: static/js/pagination.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+// The function is called to create buttons iteratively
+//  based on the page no requested 'N', total pages
+// and the entries to be listed in one page.
+function createButtons(currentPage, totalCount) {
+  let totalPages = parseInt(totalCount / pagination_size);
```
Comment (sid22) [inline static/js/pagination.js:11]: what is `pagination_size` 
make it camel case and define it

**Review by sid22 — CHANGES_REQUESTED** [review]

File: static/js/pagination.js (hunk 1 above)
Comment (RhnSaxena) [reply static/js/pagination.js:4]: They are global variables. They are defined in fhir.js file are used to track the resource in use and searchParams in use.

File: static/js/pagination.js (hunk 2 above)
Comment (RhnSaxena) [reply static/js/pagination.js:11]: It is also a global variable and was defined with that name, so I used it as it is. I will change it now.
It is in static/js/app.js .

**Review by sid22 — APPROVED** [review]

**Pull request merged**